packages/dev-frontend/src/components/Dashboard/Chart/context/ChartProvider.tsx

```diff
@@ -0,0 +1,157 @@
+import React, { useState, useEffect } from "react";
+
+import { createClient } from "urql";
+import { ChartContext } from "./ChartContext";
+
+export type BlockObject = {
+  number?: string, 
+  __typename?: string
+}
+
+export type tvlData = {
+  totalCollateral: number, 
+  blockNumber: number
+}
+
+type FunctionalPanelProps = {
+  loader?: React.ReactNode;
+};
+
+const BLOCKS_API_URL = "https://api.thegraph.com/subgraphs/name/blocklytics/goerli-blocks";
```

Comment:
should this be in a config file thats dependant on which network the user is connected to?

Reply:
solved in [feat: added apiurl in config file](https://github.com/Threshold-USD/dev/pull/46/commits/afa13a7785999a4d69031baa035f230ecae4a619)